qcodes/tests/drivers/test_MercuryiPS.py

```diff
@@ -53,6 +56,37 @@ def cylindrical_limits(x, y, z):
     mips_cl.close()
 
 
+# for testing ramp orders, we need access to the instrument communication
+
+iostream = io.StringIO()
+logger = logging.getLogger('qcodes.instrument.visa')
+logger.setLevel(logging.DEBUG)
+formatter = logging.Formatter('%(created)s - %(message)s')
+lh = logging.StreamHandler(iostream)
+logger.addHandler(lh)
+lh.setLevel(logging.DEBUG)
+lh.setFormatter(formatter)
+
+
+def get_messages(iostream: io.StringIO) -> List[str]:
```

Comment:
Could we use pytests capture log for this i.e. https://docs.pytest.org/en/latest/logging.html#caplog-fixture ?

Reply:
LOL! ^^